_posts/pl/2022-03-16-npx.md

```diff
@@ -0,0 +1,37 @@
+---
+layout:    post
+title:     "Czy wiesz, czym jest npx?"
+date:      2022-03-18 8:00:00 +0100
+published: true
+didyouknow: true
+lang: pl
+author:    mchrapkowicz
+image:     /assets/img/posts/2022-03-16-npx/keyboard.jpg
+tags:
+- npm
+- npx
+- node
+---
+
+## Czym jest npx?
+
+npx to narzędzie CLI (Command Line Interface) pozwalające wykonywać skrypty pochodzące z repozytorium npm bez zbędnej konfiguracji. Co ciekawe - można w ten sposób wywołać polecenie z pakietu niezainstalowanego na maszynie lokalnej.
```

Comment:
**NPX to przydatne** narzędzie CLI (Command Line Interface) pozwalające wykonywać skrypty pochodzące z repozytorium NPM bez zbędnej konfiguracji. Co ciekawe**,** można w ten sposób wywołać polecenie z pakietu niezainstalowanego na maszynie lokalnej.

Reply:
done